.changelog/2392.txt

```diff
@@ -1,3 +1,3 @@
-```release-note:bug
-control-plane: Always update ACL policies upon upgrade
-```
\ No newline at end of file
+```release-note:breaking-change
+control-plane: Always update ACL policies for consul-connect-injector upon upgrade. 
```

Comment:
The previous bug changelong can remain the same (we're updating all policies not just injector)

The breaking change would be: "All policies managed by consul-k8s will now be updated on upgrade. If you previously edited the policies after install, your changes will be overwritten."

Reply:
```
    ```release-note:bug
    control-plane: Always update ACL policies upon upgrade.
    ```
    
    ```release-note:breaking-change
    control-plane: All policies managed by consul-k8s will now be updated on upgrade. If you previously edited the policies after install, your changes will be overwritten.
    ```
```

Reply:
Ok thanks! That was helpful!